custom/icds_reports/reports/daily_indicators.py

```diff
@@ -0,0 +1,86 @@
+from custom.icds_reports.models import AggAwcDailyView
+from dateutil.relativedelta import relativedelta
+from io import StringIO
+from custom.icds_reports.utils import apply_exclude
+import csv
+from datetime import date
+
+
+def _get_data_for_daily_indicators(filters, domain, show_test=False):
+    queryset = AggAwcDailyView.objects.filter(
+        **filters
+    ).values('state_name', 'num_launched_awcs', 'daily_attendance_open',
+             'total_eligible_pse', 'total_attended_pse')
+    if not show_test:
+        queryset = apply_exclude(domain, queryset)
+    return queryset
+
+
+def get_daily_indicators(domain):
+    today_date = date.today()
+    filters = {
+        'aggregation_level': 1,
+        'date': today_date
+    }
+
+    filename = 'CAS_Daily_Status_{}.csv'.format(filters['date'].strftime('%Y%m%d'))
+    filters['date'] -= relativedelta(days=1)
```

Comment:
I think this would be clearer as

```python
    today = date.today()
    filename = 'CAS_Daily_Status_{}.csv'.format(today.strftime('%Y%m%d'))

    filters = {
        'aggregation_level': 1,
        'date': today - timedelta(days=1)
    }
```

Reply:
Included your comments here: https://github.com/dimagi/commcare-hq/pull/25748